src/test/java/com/ffxivcensus/gatherer/PlayerTest.java

```diff
@@ -72,72 +79,78 @@ public void testGetPlayer() throws Exception {
+        assertEquals(1, playerOne.getBitHasARRCollectors());
         //Assuming the below don't change
         assertFalse(playerOne.isHasARRArtbook());
-        assertEquals(playerOne.getBitHasArrArtbook(), 0);
+        assertEquals(0, playerOne.getBitHasArrArtbook());
         assertFalse(playerOne.isHasBeforeMeteor());
-        assertEquals(playerOne.getBitHasBeforeMeteor(), 0);
+        assertEquals(0, playerOne.getBitHasBeforeMeteor());
         assertFalse(playerOne.isHasBeforeTheFall());
-        assertEquals(playerOne.getBitHasBeforeTheFall(), 0);
+        assertEquals(0, playerOne.getBitHasBeforeTheFall());
         assertFalse(playerOne.isHasSoundtrack());
-        assertEquals(playerOne.getBitHasSoundTrack(), 0);
+        assertEquals(0, playerOne.getBitHasSoundTrack());
         assertFalse(playerOne.isHasMooglePlush());
-        assertEquals(playerOne.getBitHasMooglePlush(), 0);
+        assertEquals(0, playerOne.getBitHasMooglePlush());
 
         //Achievements
         assertTrue(playerOne.isHasAttendedEternalBond());
-        assertEquals(playerOne.getBitHasAttendedEternalBond(), 1);
+        assertEquals(1, playerOne.getBitHasAttendedEternalBond());
         assertFalse(playerOne.isHasCompletedHWSightseeing());
-        assertEquals(playerOne.getBitHasCompletedHWSightseeing(), 0);
+        assertEquals(0, playerOne.getBitHasCompletedHWSightseeing());
         assertTrue(playerOne.isHasCompleted2pt5());
-        assertEquals(playerOne.getBitHasCompleted2pt5(), 1);
+        assertEquals(1, playerOne.getBitHasCompleted2pt5());
         assertTrue(playerOne.isHasFiftyComms());
-        assertEquals(playerOne.getBitHasFiftyComms(), 1);
+        assertEquals(1, playerOne.getBitHasFiftyComms());
         assertTrue(playerOne.isHasCompletedHildibrand());
-        assertEquals(playerOne.getBitHasCompletedHildibrand(), 1);
+        assertEquals(1, playerOne.getBitHasCompletedHildibrand());
         assertTrue(playerOne.isHasEternalBond());
-        assertEquals(playerOne.getBitHasEternalBond(), 1);
+        assertEquals(1, playerOne.getBitHasEternalBond());
         assertTrue(playerOne.isHasKobold());
-        assertEquals(playerOne.getBitHasKobold(), 1);
+        assertEquals(1, playerOne.getBitHasKobold());
         assertTrue(playerOne.isHasSahagin());
-        assertEquals(playerOne.getBitHasSahagin(), 1);
+        assertEquals(1, playerOne.getBitHasSahagin());
         assertTrue(playerOne.isHasAmaljaa());
-        assertEquals(playerOne.getBitHasAmaljaa(), 1);
+        assertEquals(1, playerOne.getBitHasAmaljaa());
         assertTrue(playerOne.isHasSylph());
-        assertEquals(playerOne.getBitHasSylph(), 1);
+        assertEquals(1, playerOne.getBitHasSylph());
         assertTrue(playerOne.isHasCompletedHW());
-        assertEquals(playerOne.getBitHasCompletedHW(), 1);
+        assertEquals(1, playerOne.getBitHasCompletedHW());
+        // Currently no way to definitively tell player has completed the SB Main Scenario
```

Comment:
Myself and @Pricetx are not fond of the 'hit and miss nature' of this way of evaluating 4.0 MSQ completion, please remove this logic